test/verify/check-metrics

```diff
@@ -1318,11 +1318,13 @@ class TestMetricsPackages(packagelib.PackageCase):
 
         redis_service = redisService(m.image)
         redis_package = "valkey" if redis_service == "valkey" else "redis"
-        extra_packages = ""
+        extra_packages = []
         if redis_package == "valkey" and m.image.startswith("fedora"):
             # Fedora has a compat package that we need to
             # remove at the same time as valkey.
-            extra_packages = "valkey-compat-redis"
+            # Fedora split out valkey to more packages as
+            # well which we need to remove.
+            extra_packages += [m.execute("rpm --qf '%{NAME} ' -qa valkey-*")]
```

Comment:
This is misleading -- it's an array with a single string that can have multiple package names. Please either do `.split()`, or keep it as the simple string that it was originally. That's fine, it comes out of a shell and gets back into a shell.